frontends/mit-open/src/pages/ListDetailsPage/ItemsListing.tsx

```diff
@@ -45,21 +54,22 @@ const ItemsListingViewOnly: React.FC<{
 }
 
 const ItemsListingSortable: React.FC<{
+  listType: NonNullable<ItemsListingProps["listType"]>
```

Comment:
```suggestion
  listType: ItemsListingProps["listType"]
```

it's already nonnullable.